frontend/src/app/components/header/Header.component.jsx

```diff
@@ -60,12 +55,8 @@ const Header = ({ loading, signIn }) => {
                 }
                 signOut={signoutFromSongbook} // not used but defined so that GammaActions doesn't complain
                 size={{ width: "min-content" }}
-                frontendUrl={
-                    process.env.NODE_ENV === "development"
-                        ? "http://localhost:3000"
-                        : "https://gamma.chalmers.it"
-                }
-                backendUrl={backendUrl}
+                frontendUrl={process.env.REACT_APP_GAMMA_FRONTEND_URL}
+                backendUrl={process.env.REACT_APP_GAMMA_BACKEND_URL}
```

Comment:
This won't work. It's pain as well. You have to do this with build arguments, but it's not worth it.

Reply:
I have fixed build arguments now. It may have taken some time but it's pretty neat

Reply:
Have tested with the dockerfile for dev and prod. Both pass their environment variables forward to react.

Reply:
it is now in production and works as expected